frontend/packages/kubevirt-plugin/src/selectors/config-map/sc-defaults.ts

```diff
@@ -35,12 +35,12 @@ export const getDefaultSCVolumeMode = (
     return volumeMode;
   }
 
-  return storageClassName === 'local-sc' ? VolumeMode.FILESYSTEM : VolumeMode.BLOCK;
+  return storageClassName === 'local-sc' ? VolumeMode.BLOCK : VolumeMode.FILESYSTEM;
```

Comment:
I talked to Alex Wels on the storage team. He said that if nothing is returned from the config map then we can default to FileSystem and RWO since every storage class should support those settings. So, we can drop the check for `local-sc` and just return FileSystem.

Reply:
good to know